backend/tiggle-root/tiggle/src/main/kotlin/com/side/tiggle/domain/reaction/service/ReactionServiceImpl.kt

```
@@ -25,25 +29,41 @@ class ReactionServiceImpl(
-                receiverId = receiverId,
                 senderId = senderId,
-                type = dto.type,
+                receiverId = receiverId
             )
-                ).apply {
-                type = dto.type
-            }
-
-        reactionRepository.save(reaction)
+            reactionRepository.save(newReaction)
+        } catch (e: DataIntegrityViolationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)